database/user.go

```diff
@@ -349,6 +349,7 @@ func (db *DB) UserCreate(ctx context.Context, emailAddr, pass, sub string, tier
 		Sub:                              sub,
 		Tier:                             tier,
 	}
+	// TODO This part can race and create multiple accounts with the same email, unless we add DB-level uniqueness restriction.
```

Comment:
we should definitely have a unique index on the user's email field, do they all have an email? can only do that if they do obviously, that's probably why it's not added yet?

Reply:
It was there, we removed it. See below.